src/App.js

```diff
@@ -10,10 +10,33 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedTab: tabs[0],
+  }
+
+  switchTabs = (tabId) => {
+    this.setState({
+      selectedTab: tabs.find(tab => tab.id === tabId),
+    });
+  }
+
+  render() {
+    const { selectedTab } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          {`Selected tab is ${selectedTab.title}`}
+        </h1>
+        <Tab
+          tabs={tabs}
+          selectedTab={selectedTab}
```

Comment:
```suggestion
          selectedTabId={selectedTab.id}
```

> Pass the `selectedTabId` as a prop to the Tabs, the specified tab should be selected if possible (otherwise the first tab is selected)

You are passing the whole tabs array. You may pass only id and find needed tab there by id